lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -106,11 +106,9 @@
 
     <div
         *ngSwitchCase="'clickableTemplate'"
-        role="button"
         class="adf-textitem-clickable"
         [ngClass]="{ 'adf-property-read-only': !isEditable }"
         [attr.data-automation-id]="'card-textitem-toggle-' + property.key"
-        tabindex="0"
```

Comment:
Those changes give eslint issue: 
<img width="1001" height="480" alt="image" src="https://github.com/user-attachments/assets/046dc36b-c886-4227-84f8-f90c4fe917e4" />

Reply:
maybe worth refactoring this to be an unstyled button instead?